src/languages/swift.js

```diff
@@ -98,13 +111,72 @@ export default function(hljs) {
   };
   SUBST.contains = [NUMBER];
 
+  // https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#ID418
+  const operatorHead = either(
+    /[/=\-+!*%<>&|^~?]/,
+    /[\u00A1–\u00A7]/,
+    /[\u00A9\u00AB]/,
+    /[\u00AC\u00AE]/,
+    /[\u00B0\u00B1]/,
+    /[\u00B6\u00BB\u00BF\u00D7\u00F7]/,
+    /[\u2016–\u2017]/,
+    /[\u2020–\u2027]/,
+    /[\u2030–\u203E]/,
+    /[\u2041–\u2053]/,
+    /[\u2055–\u205E]/,
+    /[\u2190–\u23FF]/,
+    /[\u2500–\u2775]/,
+    /[\u2794–\u2BFF]/,
+    /[\u2E00–\u2E7F]/,
+    /[\u3001–\u3003]/,
+    /[\u3008–\u3020]/,
+    /[\u3030]/
+  );
+  const operatorCharacter = either(
+    operatorHead,
+    /[\u0300–\u036F]/,
+    /[\u1DC0–\u1DFF]/,
+    /[\u20D0–\u20FF]/,
+    /[\uFE00–\uFE0F]/,
+    /[\uFE20–\uFE2F]/,
+    // TODO: The following characters are also allowed, but the regex doesn't work as intended.
+    // For example, it also matches -10 as an operator.
+    // /[\u{E0100}–\u{E01EF}]/u
+  );
+  const OPERATOR = {
+    className: 'operator',
```

Comment:
Are these operators or keywords?

Reply:
Is this question about `as? as! try? try!`?

If so, the grammar specifies them as operators and keywords at the same time, so I'm highlighting them as keywords, which is what other grammars do too, although most don't bother highlighting the question or exlamation mark. I preferred to include that in the match, otherwise it would be matched as a postfix operator.

Reply:
If they are keywords then you should remove "operator", it's not serving any purpose.

Reply:
OK, I'll add them to list of "special" keywords (like the ones that have parentheses) instead, so I don't have to change the keywords $pattern to allow question marks or exclamation points.

My goal here was to group these keywords under the "operator" umbrella as well, in case a future grammar says "any operator is allowed here", which would then also include these keywords.

Reply:
> My goal here was to group these keywords under the "operator" umbrella as well, in case a future grammar says "any operator is allowed here", which would then also include these keywords.

I think you may be over thinking that. :)